nimbus.fml.yaml

```diff
@@ -33,6 +33,23 @@ features:
             "contile-top-sites": false,
           }
         }
+  onboarding:
+    description: The homescreen that the user goes to when they press home or new tab.
+    variables:
+      sections-enabled:
+        description: "This property provides a lookup table of whether or not the given section should be enabled."
+        type: Map<OnboardingSection, Boolean>
+        default:
+          {
+            "sync-cfr": true
```

Comment:
I think this should be `false`, until we enable the feature completely, meanwhile we want to enable it only we get the value from nimbus server.

Reply:
If we leave it on, it will signal that nimbus is indicating us that functionality should be ON unless we are in nightly. As we are introducing this new functionality the goal is that feature is OFF unless the nimbus server indicate so, that will allow us to turn the feature ON  for some users and see how they interact with it. And later made a decision if we want it ON for user then we can enable this.